Ginger/GingerCoreCommon/Repository/BusinessFlowLib/BusinessFlow.cs

```diff
@@ -61,6 +65,17 @@ public BusinessFlow(string sName)
             Activities.CurrentItem = a;
             CurrentActivity = a;
             AllowAutoSave = true;
+            _stopwatch = new Stopwatch();
```

Comment:
Remove it and use it from start timer method

Reply:
handled